lib/govuk_sli_collector/publishing_latency_sli/logit_search.rb

```diff
@@ -0,0 +1,66 @@
+require "rest-client"
```

Comment:
It turns out that I forgot to `require "json"` here, so I'm just going to force-push a fix for that in a second :adhesive_bandage: